benchmarks/util.py

```diff
@@ -76,11 +77,18 @@ def move_to_device(item, device, torch_xla2: bool = False):
     move_to_device_func = lambda t: jax.device_put(
         torch_xla2.tensor.t2j(t), device)
   else:
-    move_to_device_func = lambda t: t.to(device)
+
+    def move_to_device_func(tensor: torch.Tensor) -> torch.Tensor:
+      # If `tensor` is an XLA tensor, first move it to CPU. We need to do
+      # that if we want to move the tensor to, say, CUDA.
+      if tensor.device.type == "xla":
+        return tensor.cpu().to(device)
+      return tensor.to(device)
```

Comment:
Is this because of the `CUDA fallback` you implemented recently?

Reply:
Not really. It's just that, if we want to do XLA -> CUDA, we need to go through the CPU (unless we use DLPack). Previously, I think we only did CUDA -> XLA, but moving to CPU explicitly.